Resources/Textures/Objects/Misc/nukedisk.rsi/meta.json

```diff
@@ -1,20 +1,14 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/32a2e3e4ceda3ef442c1496cbc6c6a896d684702",
+  "copyright": "Taken from a Unicode Character “℮” Estimated Symbol (U+212E)",
```

Comment:
This should mention what font it's from and license should be updated accordingly.

Reply:
What should be put in `license` then? The font is licensed under Apache 2.0, but if I put it into the `license` field, RSI Validator says it's not one of the CC-* ones

Reply:
"Custom" and mention that the font is Apache 2.0 in copyright field.

Reply:
Custom doesn't seem to work either